CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
 
   - Major/Minor/Patch: This is an example changelog item, usually can be the commit message.
   - Append new items to make git merging easier.
+  - Minor: Implement updatable `value` API on Select component
```

Comment:
This feels like a breaking (major) change, since it has potential to break established implementations.

Reply:
That's a good question. 🤔 

a. The developer never updates the `value` prop on re-render -- this is a minor
b. The developer updates the `value` prop on re-render due to internal factors (i.e. `onChange` callback) -> component sets value (theoretically, the value is the same thus the state update effects do not run) -- this is a minor
c. The developer updates the `value` prop on re-render due to external factors (e.g. server updates) -> component sets value (and re-render to match the new value) -> the `onChange` callback is ran -> user code runs which was previously not ran -- this does seem like a major change

This brings up a good question:

- Do we want to run an `onChange` callback when the props change? Theoretically, if the props are driving a change then it means the user already handle the effects of the change. Instead, I think our `onChange` shouldn't call `onChange` when it matches the prop. 🤔 

I'm try a commit, see how it plays out in bigmaven.

Reply:
Done. It works pretty I think. It will build against bigmaven just in case.